website/static/js/localizeLastUpdatedDate.js

```diff
@@ -0,0 +1,43 @@
+function waitForElementToChange(selector) {
+    return new Promise(resolve => {
+        const observer = new MutationObserver(() => {
+            const element = document.querySelector(selector)
+            if (element) {
+                resolve(element);
+                observer.disconnect();
+            }
+        });
+
+        observer.observe(document.body, {
+            characterData: true,
+            childList: true,
+            subtree: true
+        });
+    });
+}
+
+function localizeTimeElement(element) {
+    if (element instanceof HTMLTimeElement && element.hasAttribute('datetime')) {
+        const iso8601 = element.getAttribute('datetime');
+        const date = new Date(iso8601);
+        element.textContent = date.toLocaleDateString(undefined, {
+            year: "numeric",
+            month: "2-digit",
+            day: "2-digit",
```

Comment:
Why force `DD`, `MM`, and `YYYY`? My original issue was that the date format wasn't respecting the users locale and this kind of does the same thing.

Reply:
Yes I wanted to force leading zeros in date format and still use locale=undefined. I think you're right, let the browser decide.